activerecord/lib/rails/generators/active_record/application_record/application_record_generator.rb

```diff
@@ -16,10 +16,10 @@ def create_application_record
 
         def application_record_file_name
           @application_record_file_name ||= if namespaced?
-                                              "app/models/#{namespaced_path}/application_record.rb"
-                                            else
-                                              "app/models/application_record.rb"
-                                            end
+            "app/models/#{namespaced_path}/application_record.rb"
+          else
+            "app/models/application_record.rb"
+          end
```

Comment:
I see! That is a wonderful solution. I updated it.

FYI, although the timing is unknown, Code Climate seems to be working to bump RuboCop to 0.49.1.

https://github.com/codeclimate/codeclimate-rubocop/issues/99#issuecomment-322381645

I think @matthewd suggestion is good now. Thank you.